src/style.scss

```
@@ -64,15 +64,23 @@ input {
   }
 }
 
+// .inputElement:focus-within {
+//   padding-right: 1.875rem;
+// }
+
```

```suggestion
// .inputElement:focus-within {
//   padding-right: 1.875rem;
// }

```

Remove commented out code